packages/trader/src/AppV2/Components/Guide/guide.tsx

```diff
@@ -12,88 +12,106 @@ import { sendOpenGuideToAnalytics } from '../../../Analytics';
+            (a, b) => order.findIndex(item => item === a.id) - order.findIndex(item => item === b.id)
+        );
 
-    const [is_description_opened, setIsDescriptionOpened] = React.useState(false);
-    const [selected_contract_type, setSelectedContractType] = React.useState(contract_type_title);
-    const [selected_term, setSelectedTerm] = React.useState<string>('');
+        const [is_description_opened, setIsDescriptionOpened] = React.useState(is_open_by_default);
+        const [selected_contract_type, setSelectedContractType] = React.useState(contract_type_title);
+        const [selected_term, setSelectedTerm] = React.useState<string>('');
 
-    const onChipSelect = React.useCallback((id: string) => setSelectedContractType(id ?? ''), []);
+        const onChipSelect = React.useCallback((id: string) => setSelectedContractType(id ?? ''), []);
 
-    const onClose = React.useCallback(() => setIsDescriptionOpened(false), []);
+        const onClose = React.useCallback(() => setIsDescriptionOpened(false), []);
 
-    React.useEffect(() => {
-        if (show_guide_for_selected_contract) setSelectedContractType(contract_type_title);
-    }, [show_guide_for_selected_contract, contract_type_title]);
+        React.useEffect(() => {
+            if (show_guide_for_selected_contract) setSelectedContractType(contract_type_title);
+        }, [show_guide_for_selected_contract, contract_type_title]);
 
-    return (
-        <React.Fragment>
-            <Button
-                color={is_dark_mode_on ? 'white' : 'black'}
-                icon={<LabelPairedPresentationScreenSmRegularIcon key='guide-button-icon' />}
-                onClick={() => {
-                    sendOpenGuideToAnalytics(
-                        contract_type,
-                        show_guide_for_selected_contract ? 'main_trade_page' : 'trade_type_page'
-                    );
-                    setIsDescriptionOpened(true);
-                }}
-                variant={has_label ? 'secondary' : 'tertiary'}
-                key={current_language}
-            >
-                {has_label && (
-                    <Text size='sm' bold color='quill-typography__color--prominent'>
-                        <Localize i18n_default_text='Guide' />
-                    </Text>
+        React.useEffect(() => {
+            setIsDescriptionOpened(is_description_opened);
+        }, [is_open_by_default]);
+
+        return (
+            <React.Fragment>
+                {show_trigger_button && (
+                    <Button
+                        color={is_dark_mode_on ? 'white' : 'black'}
+                        icon={<LabelPairedPresentationScreenSmRegularIcon key='guide-button-icon' />}
+                        onClick={() => {
+                            sendOpenGuideToAnalytics(
+                                contract_type,
+                                show_guide_for_selected_contract ? 'main_trade_page' : 'trade_type_page'
```

Comment:
this needs to be cleaned up to 'main_trade_page' since the button wont be rendered in trade type modal

Reply:
Agree, thank you, will apply